plugins/base/src/main/kotlin/renderers/gfm/GfmRenderer.kt

```diff
@@ -0,0 +1,185 @@
+package org.jetbrains.dokka.base.renderers.gfm
+
+import org.jetbrains.dokka.base.renderers.DefaultRenderer
+import org.jetbrains.dokka.base.resolvers.DefaultLocationProvider
+import org.jetbrains.dokka.base.resolvers.LocationProviderFactory
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.plugability.DokkaContext
+import java.lang.StringBuilder
+
+
+open class GfmRenderer(context: DokkaContext) : DefaultRenderer<StringBuilder>(context) {
+
+    override fun StringBuilder.buildHeader(level: Int, content: StringBuilder.() -> Unit) {
+        buildParagraph()
+        append("${"#".repeat(level)} ")
+        content()
+        buildNewLine()
+    }
+
+    override fun StringBuilder.buildLink(address: String, content: StringBuilder.() -> Unit) {
+        append("[")
+        content()
+        append("]($address)")
+    }
+
+    override fun StringBuilder.buildList(node: ContentList, pageContext: ContentPage) {
+        buildListLevel(node, pageContext)
+        buildParagraph()
+    }
+
+    private val indent = " ".repeat(4)
+
+    private fun StringBuilder.buildListItem(items: List<ContentNode>, pageContext: ContentPage, bullet: String = "*") {
+        items.forEach {
+            if(it is ContentList) {
+                val builder = StringBuilder()
+                builder.append(indent)
+                builder.buildListLevel(it, pageContext)
+                append(builder.toString().replace(Regex("  \n(?!$)"), "  \n$indent"))
```

Comment:
```suggestion
                append(builder.toString().replace(Regex("\\s{2}\n(?!$)"), "  \n$indent"))
```
looks better IMHO, also I'd add a comment explaining why it is there

Reply:
It may look better but unfortunately it is not equal. However it may work in our scope. I will check that.